File: examples/ruby/sinatra/play-url.rb
require 'sinatra'
require 'builder'

set :port, 3000

post '/' do
  logger.info "from: #{params[:from]}"
  logger.info "to: #{params[:to]}"

  headers 'Content-Type' => "application/xml"
  xml = Builder::XmlMarkup.new(:indent => 2)
  xml.instruct!
  xml.Response{xml.Play{xml.Url("http://www.example.com/example.wav")}}


end

File: examples/nodejs/express/play-url.js
var app = require('express')();
var bodyParser = require('body-parser');
var xml = require('xml');

app.use(bodyParser.urlencoded({ extended: false }));

app.post("/", function (request, response) {
	var from = request.body.from;
	var to = request.body.to;

	console.log("from: " + from);
	console.log("to: " + to);

	response.set('Content-Type', 'application/xml');
	response.send(
		xml({ Response: [
			{ Play: [
				{ Url: "http://www.example.com/example.wav" }
			] },
		] })
	);
});

app.listen(3000);
